k8s/core/configmap/configmap_lock_v1_test.go

```diff
@@ -88,19 +89,19 @@ func TestLock(t *testing.T) {
 	fatalLockCb := func(format string, args ...interface{}) {
 		fmt.Println("\tLock timeout called.")
 		lockTimedout = true
-		err := cm.Unlock()
-		require.NoError(t, err, "Unexpected error from Unlock")
 	}
 	SetFatalCb(fatalLockCb)
 
-	// Check lock expiration
 	err = cm.Lock("id2")
 	require.NoError(t, err, "Unexpected error in lock")
 	time.Sleep(20 * time.Second)
 	require.True(t, lockTimedout, "Lock hold timeout not triggered")
 
+	err = cm.Unlock()
+	require.NoError(t, err, "Unexpected no error in unlock")
+
 	err = cm.Lock("id3")
-	require.NoError(t, err, "Lock should have expired")
+	require.NoError(t, err, "Unexpected error in lock")
```

Comment:
nit: The previous error message is more appropriate

Reply:
done